start_admin_sshd.sh

```diff
@@ -47,6 +47,26 @@ get_user_data_keys() {
     ( IFS=$'\n'; echo "${valid_keys[*]}" )
 }
 
+# Export proxy environment variables for all users' login shells
+# Match the values of the proxy environment variables given to the admin container
+set_up_proxy_env_vars() {
```

Comment:
nit: I suggest naming this to align with its actions and the role it plays - installing a profile script that included the proxy details for use by shells.

```suggestion
install_proxy_profile() {
```